lib/tasks/hathi_etas.rake

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+HATHI_DATA_HOME = '/data/hathitrust_data'
```

Comment:
I am trying now to use the `settings` present in psulib_config.rb. Maybe you can load that here? I'm not 100% sure what the "right way" of doing this would be, but once loaded you could just use `settings['hathi_overlap_path']`

Something like https://github.com/psu-libraries/psulib_traject/blob/master/lib/tasks/incrementals.rake#L13-L14 would have to be run first, then `indexer.settings` could be accessed (maybe)

welcome other ideas

Reply:
yes it makes sense to get it from indexer.settings, I will give it a try, thnx! 👍

Reply:
this is done with the last commit